### PR TITLE
log/notify on failure to report to slack as we do with connection errors

### PR DESCRIPTION
**Pull request by grosser**
got all our slack hooks silently failing with a 404 without anyone noticing ... should report that ...

### CODE REVIEW

File: plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb [hunk 1]
```diff
@@ -36,10 +36,13 @@ def deliver_message_via_webhook(webhook:, message:, attachments:)
       payload[:channel] = webhook.channel unless webhook.channel.blank?
       payload[:attachments] = attachments if attachments.present?
 
-      Faraday.post(webhook.webhook_url, payload: payload.to_json)
-    rescue Faraday::ClientError => e
-      Airbrake.notify(e)
-      Rails.logger.error("Could not deliver slack message to webhook #{webhook.webhook_url}: #{e.message}")
+      begin
```
Comment (henders) [inline plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb]: why bother with this scoped `begin`? Seems unlikely an exception would be thrown in the above 3 lines?

**Comment by henders** [comment]
👍  regardless of comment

File: plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb (hunk 1 above)
Comment (henders) [inline plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb]: And even if it was, probably still want to know about it

Comment (grosser) [inline plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb]: RuntimeError could be coming from anywhere above

**Pull request merged**